Src/FluentAssertions/Events/EventRecorder.cs

```diff
@@ -121,6 +116,20 @@ public void Reset()
                 }
             }
         }
+        IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+
+        public IEnumerator<OccurredEvent> GetEnumerator()
+        {
+            foreach (RecordedEvent @event in raisedEvents.ToArray())
```

Comment:
Should there be a lock around `raisedEvent.ToArray()`?

Reply:
No, because it's a `BlockingCollection`. Wondering if I should change the name of field to make that clearer.

Reply:
Correct.